datasets/bgt/bgt_buildings_demo.csv

```diff
@@ -1,22 +1,32 @@
-363100012156186,"[[119849.382, 485299.971], [119854.204, 485288.353], [119848.906, 485285.986], [119844.059, 485297.595], [119843.166, 485299.734], [119843.099, 485299.894], [119848.398, 485302.261], [119848.457, 485302.125], [119849.382, 485299.971]]",119843.099,485302.261,119854.204,485285.986
-363100012146254,"[[119891.063, 485282.461], [119891.0997295, 485282.4364709], [119891.138402, 485282.4151364], [119891.1787406, 485282.3971493], [119891.2204559, 485282.3826387], [119891.2632491, 485282.3717084], [119891.3068135, 485282.3644369], [119891.3508368, 485282.3608762], [119891.3950035, 485282.3610518], [119891.4389971, 485282.3649626], [119891.4825023, 485282.3725804], [119891.5252072, 485282.3838507], [119891.5668058, 485282.3986927], [119891.607, 485282.417], [119891.6474218, 485282.4398362], [119891.6856534, 485282.4661755], [119891.7213921, 485282.4958093], [119891.7543548, 485282.528503], [119891.7842804, 485282.5639977], [119891.8109321, 485282.6020122], [119891.8340986, 485282.6422456], [119891.8535967, 485282.6843792], [119891.8692719, 485282.7280793], [119891.881, 485282.773], [119891.9967396, 485282.6792328], [119892.1039277, 485282.5757985], [119892.2017595, 485282.4634737], [119892.2895007, 485282.3431016], [119892.3664923, 485282.2155859], [119892.4321564, 485282.0818842], [119892.486, 485281.943], [119892.5242866, 485281.8133696], [119892.5522704, 485281.6811318], [119892.5697785, 485281.5471043], [119892.5767026, 485281.4121155], [119892.573, 485281.277], [119892.5743231, 485281.2310737], [119892.5717441, 485281.1852009], [119892.5652818, 485281.1397123], [119892.5549827, 485281.0949362], [119892.5409211, 485281.0511956], [119892.5231984, 485281.0088059], [119892.5019426, 485280.9680731], [119892.4773069, 485280.929291], [119892.4494691, 485280.8927392], [119892.41863, 485280.8586816], [119892.3850121, 485280.8273638], [119892.3488578, 485280.7990117], [119892.310428, 485280.7738299], [119892.27, 485280.752], [119892.226051, 485280.7329844], [119892.1805926, 485280.71793], [119892.1339809, 485280.7069547], [119892.0865812, 485280.7001447], [119892.038765, 485280.6975532], [119891.9909069, 485280.6992005], [119891.9433821, 485280.7050738], [119891.8965629, 485280.715127], [119891.8508162, 485280.7292813], [119891.8065005, 485280.7474259], [119891.7639631, 485280.7694185], [119891.7235373, 485280.7950868], [119891.6855399, 485280.8242296], [119891.6502686, 485280.8566186], [119891.618, 485280.892], [119891.5119893, 485280.9914653], [119891.4150153, 485281.0997596], [119891.3278105, 485281.2160649], [119891.2510336, 485281.3395028], [119891.1852645, 485281.4691409], [119891.131, 485281.604], [119891.0887759, 485281.7425727], [119891.0587005, 485281.8842792], [119891.0409994, 485282.0280565], [119891.0358053, 485282.1728263], [119891.0431572, 485282.3175025], [119891.063, 485282.461]]",119891.0358053,485282.773,119892.5767026,485280.6975532
+bgt_type,polygon,x_min,y_max,x_max,y_min

```

Comment:
Het lijkt erop dat er meer veranderd is aan dit bestand dan alleen de column name "building_id"

Reply:
Ik had de buffer op 5m gezet ipv 10, misschien is dat het?

Reply:
Dat moet het wel zijn ja

Reply:
Heb je de "bbox code" gerunt, of heb je apart voor de 2 tiles de bgt data opgehaald en daarna samengevoegd?

Reply:
Gecheckt geplot, zijn juist.